spec/formatting.md

```diff
@@ -344,6 +356,11 @@ The resolved value of _markup_ includes the following fields:
 - The _identifier_ of the _markup_
 - The resolved _options_ values after _option resolution_.
 
+If the resolved mapping of _options_ includes any `u:` options
+supported by the implementation,
+process them as specified in the [Unicode Registry](/spec/registry/unicode.md).
+Such `u:` options MAY be removed from the resolved mapping of _options_.
```

Comment:
See above

Reply:
> Such `u:` options MAY be removed from the resolved mapping of _options_.

?
Why?

Reply:
I think @eemeli is thinking that they've "done their job", e.g. if you have:

```
.input {$foo :datetime u:locale=fr}
```

... then the implementation has read the `u:locale` option and set the locale of `$foo`'s `:datetime` invocation to French (in the formatting context). In such a case, `:datetime` doesn't need to know about this. In most cases, I think that's correct (if the function wants to know about the locale or direction or id, it can read it from the formatting context). But I could see a function or markup wanting access to the `u:id`. I can also imagine `u:xxx` options in the future where that might not be the right thing to do. Finally, this creates work for implementations (although it is only `MAY` and not required work).